lerna.json

```diff
@@ -0,0 +1,5 @@
+{
+  "npmClient": "yarn",
+  "version": "independent",
```

Comment:
Can we change to fixed versioning? We should discuss more, but I'm leaning towards a model where all our SDK packages have the same version. This'll make it easier for developers who need to pull in multiple packages and keep their versions consistent.

Reply:
Yeah, I think that makes sense. It'd be a bit nicer to have the versions truly indicate compatibility rather than having to guess and check. Is there a specific version you'd want to move to initially? Seems like we could do a version bump to `0.3.3` for everything to match the current version of the viewer, or we could go with a higher version (like `0.4.0` or something)